scripts/after-pack-hook.js

```diff
@@ -36,5 +37,17 @@ module.exports = async function (params) {
     await fs.copy(sourceFolder, destinationFolder)
   }
 
+  const distNodeModules = path.join(params.packager.info._appDir, 'node_modules')
+  const appNodeModules = path.join(outputFolder, 'node_modules')
+
+  console.log('copying ', distNodeModules, ' to', appNodeModules)
+
+  await fs.copy(distNodeModules, appNodeModules)
+
+  // const packages = await globby('packages/*/node_modules', {
+  //   cwd: params.packager.info._appDir,
+  //   onlyFiles: false,
+  // })
+
```

Comment:
keep?

Reply:
This code was moved up to line 15 above, can remove this dead code. Will fix & push up after the current CI builds since the mac binary signing takes awhile